test/commands/channels/configure.test.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2022, Nuance, Inc. and its contributors.
+ * All rights reserved.
+ *
+ * This source code is licensed under the Apache-2.0 license found in
+ * the LICENSE file in the root directory of this source tree.
+ */
+
+import { expect, test } from '@oclif/test'
+const strip = require("strip-ansi")
+
+const td = require('./channels-test-data')
+
+const testEnvData = require('../../test-data')
+const serverURL = `https://${testEnvData.server}`
+
+describe('channels:configure', () => {
+  test
+    .env(testEnvData.env)
+    .nock(serverURL, api => api
+      .put(`/v4/projects/${td.configure.flags.project}/.channel`, td.configure.expectedBody)
+      .reply(200, td.configure.expectedResult)
+    )
+    .stdout()
+    .command(['channels:configure',
+      '--project', td.configure.flags.project,
+      '--channel', td.configure.flags.channel,
+      ...td.configure.flags.modes.map((mode: string) => `--mode=${mode}`),
+      '--color', td.configure.flags.color,
+    ])
+    .it('configures a project channel', ctx => {
+      expect(ctx.stdout).to.contain(`Channel ${td.configure.expectedResult.channel.displayName} with ID ${td.configure.expectedResult.channel.id} was updated successfully.`)
+    })
+
+  test
```

Comment:
This unit test needs to change once you remove the `try/catch` mentioned earlier.